src/kemalyst-model.cr

```diff
@@ -48,10 +55,13 @@ class Kemalyst::Model
   macro process_fields
     {% name_space = @type.name.downcase.id %}
     {% table_name = SETTINGS[:table_name] || name_space + "s" %}
+    {% primary_name = PRIMARY[:name] %}
+    {% primary_type = PRIMARY[:type] %}
     # Table Name
     @@table_name = "{{table_name}}"
+    @@primary_name = "{{primary_name}}"
```

Comment:
These class variables may no longer be needed, since all instance methods have been moved inside the macro block, so we can use the macro expressions directly instead.